src/components/ProfilePage/QuestCard/QuestCard.tsx

```diff
@@ -169,7 +169,7 @@ export const QuestCard = ({
                   lineHeight={'18px'}
                   fontWeight={600}
                   sx={{
-                    padding: '8px',
+                    padding: 8,
```

Comment:
Just to make sure and please double-check: My understanding is - using numbers in sx-props will apply the actual spacing. So, if I am not mistaken, this should turn into 64px. Other than that, it is great style to use numbers instead of px and we should do whenever we can (also the code may speak different still to this point)

Reply:
Here we go: Take-away - no numbers in sx props... unfortunately

![image](https://github.com/lifinance/jumper.exchange/assets/43956540/d52a8b50-26b1-4f23-9b99-e0a620e85b22)